src/app/member/[memberIdentifier]/study/[studyIdentifier]/review/job-review-buttons.tsx

```diff
@@ -0,0 +1,46 @@
+import { StudyJob } from '@/schema/study'
+import { Button, Group } from '@mantine/core'
+import React from 'react'
+import { StudyJobStatus } from '@/database/types'
+import { useMutation, useQuery } from '@tanstack/react-query'
+import { MinimalJobInfo } from '@/lib/types'
+import { useRouter } from 'next/navigation'
+import { dataForJobAction, updateStudyJobStatusAction } from '@/server/actions/study-job-actions'
+
+export const JobReviewButtons = ({ job, decryptedResults }: { job: StudyJob; decryptedResults: string[] }) => {
+    const router = useRouter()
+    const jobInfo = useQuery({
+        queryKey: ['jobInfo', job.id],
+        queryFn: () => {
+            return dataForJobAction(job.id)
+        },
+    }).data?.jobInfo
+
+    const { mutate: updateStudyJob } = useMutation({
+        mutationFn: async ({ jobInfo, status }: { jobInfo: MinimalJobInfo; status: StudyJobStatus }) => {
+            await updateStudyJobStatusAction(jobInfo, status, decryptedResults)
```

Comment:
are ya passing decrypted results to a server action before they're approved? or am i tired